data/core/units/elves/Sharpshooter.cfg

```diff
@@ -17,7 +17,7 @@
     {AMLA_DEFAULT}
     cost=51
     usage=archer
-    description= _ "There was a legend of an elf who stopped an arrow by shooting it out of the air with one of his own. It is a testament to the skill of the elves that such a story could be taken seriously. The sharpshooters of the elves have, quite simply, mastered the art of archery."+{SPECIAL_NOTES}+{SPECIAL_NOTES_MARKSMAN}
+    description= _ "Though not a formal title, ‘Sharpshooter’ is the epithet given to the most gifted Elvish marksmen. These expert archers are capable of feats that, even by Elven standards, border on miraculous: they are precise enough to split incoming arrows midair and can loose a second arrow before the first hits, all while maintaining enough power to damage steel armor. A single volley of their arrows is enough to bring down a knight on horseback, and even heavily-armored infantry will survive few additional shots. The sharpshooters of the elves have honed their art to its highest form and are the undisputed masters of archery."+{SPECIAL_NOTES}+{SPECIAL_NOTES_MARKSMAN}
```

Comment:
Not sure I like "they are precise enough"... consider "they have the precision" instead?